cogboard-app/src/main/kotlin/com/cognifide/cogboard/config/service/WidgetRuntimeService.kt

```diff
@@ -45,10 +46,25 @@ class WidgetRuntimeService(
             widgets[id] = WidgetIndex.create(newConfig, vertx).start()
         } else {
             LOGGER.error("Widget Update / Create | " +
-                    "There is widget with no ID in configuration: $widgetConfig")
+                    "There is no widget with given ID in configuration: $widgetConfig")
+        }
+    }
+
+    private fun internalDelete(action: String, widgetConfig: JsonObject): String? {
```

Comment:
can we rename this method to smth like: `stopAndRemove`

Reply:
Sure.